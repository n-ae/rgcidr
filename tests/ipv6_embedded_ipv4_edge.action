::ffff:0:0/96
