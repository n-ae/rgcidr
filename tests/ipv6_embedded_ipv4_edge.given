::192.168.1.1
::1:192.168.1.1
::ffff:192.168.1.1
::ffff:0:192.168.1.1
2001:db8::192.168.1.1
64:ff9b::192.168.1.1
::ffff:c0a8:101
::c0a8:101
2002:c0a8:101::1
fe80::192.168.1.1
::FFFF:192.168.1.1
::ffff:0.0.0.0
::ffff:255.255.255.255
0:0:0:0:0:ffff:192.168.1.1
0000:0000:0000:0000:0000:ffff:192.168.1.1
2001:db8:85a3::8a2e:370:192.168.1.1
::ffff:127.0.0.1
invalid::192.168.1.1::format
