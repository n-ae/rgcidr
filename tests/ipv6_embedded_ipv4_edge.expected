::ffff:192.168.1.1
::ffff:c0a8:101
::FFFF:192.168.1.1
::ffff:0.0.0.0
::ffff:255.255.255.255
0:0:0:0:0:ffff:192.168.1.1
0000:0000:0000:0000:0000:ffff:192.168.1.1
::ffff:127.0.0.1
